exo-add/src/cli.ls

```diff
@@ -82,9 +82,8 @@ function get-existing-services services
 function parse-command-line command-line-args
   data = {}
   questions = []
-  service-role = command-line-args["role"]
-  entity-name = command-line-args["_"]
-  service-type = command-line-args["type"]
+  service-role = command-line-args["service-role"]
+  service-type = command-line-args["service-type"]
   author = command-line-args["author"]
   template-name = command-line-args['template']
```

Comment:
@trushton thanks for fixing this. Do the tests run locally on your machine? On my machine they fail now.